packages/excalidraw/components/App.tsx

```diff
@@ -1950,6 +1950,10 @@ class App extends React.Component<AppProps, AppState> {
         frameElement,
         data: { status: "done", html: parsedHtml },
       });
+
```

Comment:
What's the point of these lines?

Reply:
@mtolmacs Are you referring for the extra line I have inserted between line 1952 and 1954?

Reply:
No, I was referring to this addition with setState. Why did you add that?

Reply:
@mtolmacs it is being used to update the selected Element ID after new frame is created, so the newly pasted frame gets selected

Reply:
This might be a misunderstanding, the `frameElement` here is something unrelated, it has to do with the frame tool.

Reply:
@mtolmacs oh yeah, you're correct. Thankyou for pointing our
Pushing the changes